### ISSUE TITLE
Uncaught TypeError: Cannot read property 'innerHTML' of null

### ISSUE THREAD

**Issue by flybyray** [dhanasingh/redmine_wktime#149]
https://github.com/dhanasingh/redmine_wktime/blob/a772f94db8755ed820b594ef68d89f0e03a3099b/assets/javascripts/wkstatus.js#L25

```
wkstatus.js?1527592533:25 Uncaught TypeError: Cannot read property 'innerHTML' of null
    at HTMLDocument.<anonymous> (wkstatus.js?1527592533:25)
    at j (jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2)
    at Object.fireWith [as resolveWith] (jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2)
    at Function.ready (jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2)
    at HTMLDocument.J (jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2)
(anonymous) @ wkstatus.js?1527592533:25
j @ jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2
fireWith @ jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2
ready @ jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2
J @ jquery-1.11.1-ui-1.11.0-ujs-3.1.4.js?1523104209:2

```

**Comment by dhanasingh** [comment]
could be a null pointer issue, we will take a look

**Comment by karthickmadhiazhagan** [comment]
We have added your fix in v3.3
